cypress/integration/search.spec.js

```diff
@@ -17,6 +17,14 @@ describe('Search', () => {
     cy.url().should('include', 'from=10');
   });
 
+  it('The user should see formulas correctly formatted with MathJax', () => {
+    cy.get(`[data-testid=${dataTestId.startPage.searchResultItem}]`).eq(0).should('not.contain', '$');
+    cy.get(`[data-testid=${dataTestId.startPage.searchResultItem}]`)
+      .eq(0)
+      .get('mjx-container')
+      .should('have.length', 4);
```

Comment:
Jeg trodde dette skulle være det samme, men det er det tydeligvis ikke... Ser du hva som er feil @Eiriknil ?
```suggestion
    cy.get(`[data-testid=${dataTestId.startPage.searchResultItem}]`).eq(0).within(() => {
      cy.should('not.contain', '$');
      cy.get('mjx-container').should('have.length', 4);
    }
```